x-pack/plugins/cases/public/components/cases_context/index.tsx

```diff
@@ -30,16 +45,19 @@ export const CasesProvider: React.FC<{ value: CasesContextProps }> = ({
   value: { owner, userCanCrud, basePath = DEFAULT_BASE_PATH, features = {} },
 }) => {
   const { appId, appTitle } = useApplication();
-  const [value, setValue] = useState<CasesContextStateValue>(() => ({
-    owner,
-    userCanCrud,
-    basePath,
-    /**
-     * The empty object at the beginning avoids the mutation
-     * of the DEFAULT_FEATURES object
-     */
-    features: merge({}, DEFAULT_FEATURES, features),
-  }));
+  const [state, dispatch] = useReducer(casesContextReducer, initialCasesContextState());
+  const [baseValue, setBaseValue] = useState<Omit<CasesContextStateValue, 'state' | 'dispatch'>>(
+    () => ({
+      owner,
+      userCanCrud,
+      basePath,
+      /**
+       * The empty object at the beginning avoids the mutation
+       * of the DEFAULT_FEATURES object
+       */
+      features: merge({}, DEFAULT_FEATURES, features),
+    })
+  );
```

Comment:
I think the distinction between `baseValue` and `value` is no longer needed now that we don't have the `state` in it. The `dispatch` can be placed here in the initialization directly, and then we can rename this `baseValue` back to `value`. Its type will be `CasesContextStateValue` directly again, and the `useMemo` down at line 78 can be removed.

Reply:
Just pushed that change :)